redbox/storage/storage_handler.py

```diff
@@ -48,22 +45,22 @@ def read_items(self, item_uuids: list[UUID], model_type: str):
         pass
 
     @abstractmethod
-    def update_item(self, item_uuid: UUID, item: PersistableModel):
+    def update_item(self, item: PersistableModel):
```

Comment:
do we even need this base class anymore?

Reply:
Mainly here to provide a basis for any other departments or companies who want to add in their own DB of choice.